main.js

```diff
@@ -0,0 +1,48 @@
+const books = [];
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const button = document.querySelector('.btn');
+
+// REMOVE BOOK FUNCTION
+function deleteBook(el) {
+  if (el.classList.contains('remove')) {
+    el.parentElement.parentElement.remove();
+  }
+}
+
+// ADD BOOKS TO TABLE LIST
+function addBookList() {
+  const list = document.querySelector('.book-list');
+
+  const row = document.createElement('tr');
+  // RETRIEVE THE BOOK FROM LOCAL STORAGE IN JSON FORMAT
+  window.localStorage.getItem('book');
+  // CONVERT THE JSON FORMAT BACK TO AN OBJECT
+  JSON.parse(window.localStorage.getItem('book'));
+
+  row.innerHTML = `
+                      <td>title: ${title.value}</td>
+                      <td>Author: ${author.value}</td>
+                      <td><button type="submit" class="remove">Remove</button></td>`;
+
+  list.appendChild(row);
+}
+
+// ADD INPUT VALUE
+button.addEventListener('click', () => {
+  books.push({ title: title.value, author: author.value });
+
+  addBookList();
+  window.localStorage.setItem('book', JSON.stringify(books));
```

Comment:
Very good job implementing the `LocalStorage` API to store the `books` array after adding a new book, 💯  

Sadly when I refresh the page, all the books that I created are not there anymore, which means that is something wrong with retrieving your variable from the `LocalStorage` at the beginning of your App. 

Kindly give it another look. 👀

Reply:
* Done 👍